.github/workflows/functional.yml

```diff
@@ -0,0 +1,51 @@
+name: Functional
```

Comment:
Can we have better name for this?

Reply:
Something referring to IRSO would be enough to help distinguish it, maybe simply "IrSO functional test" ?

Reply:
Ah, you want to see IrSO mentioned? Great, will update.